content/cards/organizer/1focusonproblem.md

```diff
@@ -1,5 +1,10 @@
 ### First, Focus on the problem
 
-Civic hackathons work best when focused on a specific problem set. The best way to highlight these problems is to bring in a subject matter expert who deals with the problem on an everyday basis. What to do a hackathon about crime? Better call the police department. Want to do something around housing? Recruit somebody from the local housing authority or somebody who runs a shelter. When trying to think about the needs and challenge of the front line, there’s no substitute for somebody who actually works in the front line to speak at your event.
+Civic hackathons work best when focused on a specific problem set. The best way to highlight these problems is to bring in a subject matter expert who deals with the problem on an everyday basis. Want to do a hackathon about crime? Better call the police department. Want to do something around housing? Recruit somebody from the local housing authority or somebody who runs a shelter. When trying to think about the needs and challenge of the front line, there’s no substitute for somebody who actually works in the front line to speak at your event.
 
-You also shouldn’t wait until the day of the event to provide your participants with resources. By providing a resource list ahead of time (like this one we put together for the Chicago Police Department Safer Communities Hackathon) it helps get participants better prepare for the event. Hackathons may have challenge components, but that doesn’t mean everyone absolutely has to start from scratch.
+You also shouldn’t wait until the day of the event to provide your participants with resources. By providing a resource list ahead of time, it helps get participants better prepare for the event. Hackathons may have challenge components, but that doesn’t mean everyone absolutely has to start from scratch.
+
+- [ ]  Decide on a problem set or general problem context 
+- [ ] Recruit subject matter experts who deal with this context on a daily basis to help define problems that are particularly painful 
```

Comment:
this content is duplicated in card 4, https://github.com/waffleio/hackshop/blob/master/content/cards/organizer/4RecruitStakeholders.md. Maybe move some of the checklist there?